packages/ComponentLibrary/src/components/ProfileModal/UserProfile.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {
+  userProfileStyles,
+  profileImageContainerStyles,
+  profileImageStyles,
+  svgContainerStyles,
+  profileDetailsStyles,
+  nameStyles,
+  emailStyles,
+  logoutButtonStyles,
+} from './ProfileModal.styles';
+import profilePicture from '../../assets/images/profile_picture.png';
+import BackgroundGradient from '../../assets/images/backgroundGradient.svg';
+import Logout from '@mui/icons-material/Logout';
+
+const UserProfile: React.FC = () => {
+  const user = {
```

Comment:
It's a mock.
It doesn't have to be placed here.
Move to a mock file inside this folder